release/src/main/python-release/python_release_automation_utils.sh

```diff
@@ -78,22 +78,18 @@ function get_version() {
 #   BEAM_PYTHON_SDK*
 # Arguments:
 #   $1 - SDK type: tar, wheel
-#   $2 - python interpreter version: python2.7, python3.5, ...
+#   $2 - python interpreter version: python3.7, python3.8, ...
 #######################################
 function download_files() {
   if [[ $1 = *"wheel"* ]]; then
-    if [[ $2 == "python2.7" ]]; then
-      BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp27-cp27mu-manylinux1_x86_64.whl"
-    elif [[ $2 == "python3.5" ]]; then
-      BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp35-cp35m-manylinux1_x86_64.whl"
-    elif [[ $2 == "python3.6" ]]; then
+    if [[ $2 == "python3.6" ]]; then
       BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp36-cp36m-manylinux1_x86_64.whl"
     elif [[ $2 == "python3.7" ]]; then
       BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp37-cp37m-manylinux1_x86_64.whl"
     elif [[ $2 == "python3.8" ]]; then
-      BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp38-cp38m-manylinux1_x86_64.whl"
+      BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp38-cp38-manylinux1_x86_64.whl"
     elif [[ $2 == "python3.9" ]]; then
-      BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp39-cp39m-manylinux1_x86_64.whl"
+      BEAM_PYTHON_SDK_WHL="apache_beam-$VERSION*-cp39-cp39-manylinux1_x86_64.whl"
```

Comment:
Is dropping the `m` here intentional?

Reply:
Yes, the format of wheels changed in 3.8. I expect it to continue in 3.9, although haven't seen 3.9 wheels yet.

From https://wheel.readthedocs.io/en/stable/news.html:
```
0.33.5 (2019-08-17)

Don’t add the m ABI flag to wheel names on Python 3.8 (PR by rdb)
Updated MANIFEST.in to include many previously omitted files in the sdist
```

Reply:
:+1: thanks